website/docs/clients/reactnative.md

```diff
@@ -160,23 +160,26 @@ We need to call `ups.init` to initialize our library. This method takes one para
 
 With an initialized library, we may now register with the Unified Push server. Registration should be called every time you load your application and any time you wish to change your user's alias or category. As a reminder, an alias is an identifier for your user shared among their devices (such as an email or username), and categories are subscriptions to messages that are sent to groups of users. The following example registers a user with an alias subscribed to two categories.
 
-```
+```javascript
 ups.register({
       "alias":"rnAlias",
       "categories":["cat1", "cat2"]
 }).then(()=>{
   //You are registered, inform your app
 })
+```
 
 ## Register Message Handlers
 
 A message handler is a function that takes a single parameter, the message. This message is your push notification data. You can register the message handler with the following code : 
 
-```
+```javascript
 const ups = new RNUnifiedPush();
 ups.registerMessageHandler(message => console.log("You have received a push message.",  JSON.stringify(message)));
 ```
 
+You will see different push objects when you run your application on Android vs when you run it on iOS. Message handlers can switch on the `aps` property of the message. This value is set on iOS and is not set on Android. 
+
```

Comment:
I believe that instead of switch on the `aps` property, would be better to use this: https://reactnative.dev/docs/platform-specific-code

WDYT?